slides/03-what-makes-a-model.qmd

```diff
@@ -150,9 +150,9 @@ decision_tree() %>%
 
 . . .
 
-::: smaller
+<br></br>
+
 All available models are listed at <https://www.tidymodels.org/find/parsnip/>
-:::
```

Comment:
The smaller thing here doesn't seem to be doing anything, but it does look like we need a little more spacing between the code chunk and this bit of prose so I used a manual break